docs/DafnyRef/Attributes.md

```diff
@@ -344,136 +326,220 @@ recursion was explicitly requested.
+     
+Skip verification of an implementation altogether
+
+#### 22.2.2.3. `{:vcs_max_cost N}`
+#### 22.2.2.4. `{:vcs_max_splits N}`
+#### 22.2.2.5. `{:vcs_max_keep_going_splits N}`
+Per-implementation versions of
+`/vcsMaxCost`, `/vcsMaxSplits` and `/vcsMaxKeepGoingSplits`.
+
+#### 22.2.2.6. `{:selective_checking true}`
+Turn all asserts into assumes except for the ones reachable from
+assumptions marked with the attribute `{:start_checking_here}`.
+Thus, `assume {:start_checking_here} something;` becomes an inverse
+of `assume false;`: the first one disables all verification before
+it, and the second one disables all verification after.
+
+#### 22.2.2.7. `{:priority N}`
+Assign a positive priority 'N' to an implementation to control the order
+in which implementations are verified (default: N = 1).
+
+#### 22.2.2.8. `{:id <string>}`
+Assign a unique ID to an implementation to be used for verification
+result caching (default: "<impl. name>:0").
+
+#### 22.2.2.9. `{:timeLimit N}`
+Set the time limit for a given implementation.
+
+### 22.2.3. Verification attributes on functions
+
+#### 22.2.3.1. `{:builtin "spec"}`
+#### 22.2.3.2. `{:bvbuiltin "spec"}`
+Rewrite the function to built-in prover function symbol 'fn'.
+
+#### 22.2.3.3. `{:inline}`
+`{:inline}` or `{:inline true}` expands a function according to its definition before going to the prover.
+
+#### 22.2.3.4. `{:never_pattern true}`
+Terms starting with this function symbol will never be
+automatically selected as patterns. It does not prevent them
+from being used inside the triggers, and does not affect explicit
+trigger annotations. Internally it works by adding `{:nopats ...}`
+annotations to quantifiers.
+
+#### 22.2.3.5. `{:identity}`
+If the function annotated with `{:identity}` or `{:identity true}` has 1 argument and the use of it has type X->X for
+some X, then the abstract interpreter will treat the function as an
+identity function.  Note, the abstract interpreter trusts the
+attribute--it does not try to verify that the function really is an
+identity function on the abstract domain.
 
+### 22.2.4. Verification attributes on variables
+
+#### 22.2.4.1. `{:existential true}`
+Marks a global Boolean variable as existentially quantified. If
+used in combination with option `/contractInfer` Boogie will check
+whether there exists a Boolean assignment to the existentials
+that makes all verification conditions valid.  Without option
+`/contractInfer` the attribute is ignored.
+
+### 22.2.5. Verification attributes on assert statements {#sec-verification-attributes-on-assert-statements}
+
+To understand how to tweak verification,
+it is first useful to understand how Dafny verifies functions and methods.
+
+For every method, Dafny extracts _assertions_, as follows:
+
+* any explicit [`assert` statement](#sec-assert-statement) is _an assertion_.
+* A consecutive pair of lines in a [`calc` statement](#sec-calc-statement) forms _an assertion of their equality_.
```

Comment:
Not necessarily equality, you can specify a different chaining operator such as `<=`.